src/main/java/com/akukhtin/springbootapplication/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.akukhtin.springbootapplication.controller;
+
+import com.akukhtin.springbootapplication.dto.UserDto;
+import com.akukhtin.springbootapplication.dto.util.UtilDto;
+import com.akukhtin.springbootapplication.exception.UserNotFoundException;
+import com.akukhtin.springbootapplication.model.User;
+import com.akukhtin.springbootapplication.service.UserService;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/add")
```

Comment:
```suggestion
    @PostMapping
```
з точки зору правильної REST концепції нам не треба додавати зайвий path для Create операції